.eslintrc.js

```diff
@@ -6,8 +6,14 @@ module.exports = {
     'eslint-config-prettier',
     'plugin:jsdoc/recommended',
     'plugin:react-hooks/recommended',
+    'plugin:@typescript-eslint/recommended',
+  ],
+  ignorePatterns: [
+    'lib/**/*.js.',
+    'lib/*.js',
+    'src/actions.ts',
```

Comment:
why ignore action.ts and store.ts ?

Reply:
Because in those file i use a typescript 4.^ feature and when i did this pr, the eslint version wasn't compatible with the version of typescript used

I have updated the version of eslint parse to a new one that is compatible